### PR TITLE
Slighlty reduce calls to dynamo

### PR DESCRIPTION
**Pull request by alexduf**
This is an attempt to reduce the amount of calls we make to dynamo. The only interesting commit is 1069e9b the rest is log handling and syntactic sugar.

I don't think this will be enough to solve the timeout issue though, just a quick-win I could find when reading the code.

### CODE REVIEW

File: src/main/scala/com/gu/mobile/content/notifications/Lambda.scala [hunk 1]
```diff
@@ -56,19 +56,17 @@ object Lambda extends NotificationsDebugLogger {
...
+            case EventPayload.Content(content) =>
+              logDebug(s"Handle content update ${content.id}")
+              val send = sendNotification(content)
+              Future.successful(send)
+            case EventPayload.RetrievableContent(content) =>
+              logDebug(s"Handle retrievable content or not: ${content.id}")
+              handleRetrievableContent(content)
+            case UnknownUnionField(e) =>
+              logDebug(s"Unknown event payload $e. Consider updating capi models")
+              Future.successful(false)
```
Comment (NathanielBennett) [inline src/main/scala/com/gu/mobile/content/notifications/Lambda.scala]: Nice. My bad

File: src/main/scala/com/gu/mobile/content/notifications/Lambda.scala [hunk 2]
```diff
@@ -78,16 +76,21 @@ object Lambda extends NotificationsDebugLogger {
   }
 
   private def sendNotification(content: Content): Boolean = {
```
Comment (NathanielBennett) [inline src/main/scala/com/gu/mobile/content/notifications/Lambda.scala]: Yeah I left all this debug in because I thought I might need it. My bad II

**Comment by NathanielBennett** [comment]
:+1:

**Comment by alexduf** [comment]
Following our chat, I actually think that will be enough to handle the load of a batch tagging operation.

**Pull request merged**